package.json

```diff
@@ -28,7 +28,12 @@
     "eslint-config-airbnb-base": "^14.2.0"
   },
   "peerDependencies": {
-    "@typescript-eslint/eslint-plugin": "^3.6.1"
+    "@typescript-eslint/eslint-plugin": "^3.6.1",
+    "eslint": "^5.16.0 || ^6.8.0 || ^7.2.0",
+    "eslint-plugin-import": "^2.21.2",
```

Comment:
Curious where this version came from?

In both [eslint-config-airbnb](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb/package.json#L79) and [eslint-config-airbnb-base](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/package.json#L69) they reference `^2.22.0`

Reply:
I got that version from the warning logs when installing with Yarn V2 PNP.

Reply:
Looks like the peer dependency version was updated pretty recently in `eslint-config-airbnb`. Maybe the warning log you referenced was slightly outdated. Can you update this line `^2.22.0`?

Reply:
This will be a bit hard to maintain. To mimic other libraries' peer dependencies will just go out of sync with time. I think it's cleaner to deal with this problem to decouple this library's dependencies from airbnb as much as possible and instead provide guidance on how to deal with these. 
An alternative approach is proposed here https://github.com/iamturns/eslint-config-airbnb-typescript/pull/201